### PR TITLE
[ODS-4630] Plugin folder should move to the API

### PR DESCRIPTION
**Pull request by jamessmckay**

### CODE REVIEW

File: Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1 [hunk 1]
```diff
@@ -171,12 +171,14 @@ function Initialize-DevelopmentEnvironment {
 }
 
 function Invoke-PluginFolderScript {
-    $settings = Get-DeploymentSettings
-    $pluginFolder = (Get-RepositoryResolvedPath "Plugin").Path
+    Invoke-Task -name $MyInvocation.MyCommand.Name -task {
+        $settings = Get-DeploymentSettings
+        $pluginFolder = (Get-RepositoryResolvedPath "Plugin").Path
```
Comment (tbmeador-dlp) [inline Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1:176]: should this be using the plugin property from the appsettings.json?

File: logistics/scripts/modules/plugin/plugin-source.psm1 [hunk 2]
```diff
@@ -72,7 +72,7 @@ function Get-PluginScriptsForPackaging {
 
     $pluginFolder = (Get-RepositoryResolvedPath "Plugin").Path
```
Comment (tbmeador-dlp) [inline logistics/scripts/modules/plugin/plugin-source.psm1:73]: should this be using the plugin:folder property from the appsettings.json?

**Review by gmcelhanon — APPROVED** [review]

File: Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1 (hunk 1 above)
Comment (tbmeador-dlp) [reply Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1:176]: also it looks like `Invoke-PluginFolderScript` is duplicated both in initdev and the deployment scripts but was already implemented in the plugin.psm1 script as `Get-Plugins`. we should probably clean this up.

**Review by gmcelhanon — APPROVED** [review]

File: Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1 (hunk 1 above)
Comment (tbmeador-dlp) [reply Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1:176]: created [ODS-4630](https://tracker.ed-fi.org/browse/ODS-4630) to address these issues

**Pull request merged**